packages/vm/src/state/interface.ts

```diff
@@ -33,6 +33,7 @@ export interface StateManager {
   accountExists(address: Address): Promise<boolean>
   cleanupTouchedAccounts(): Promise<void>
   clearOriginalStorageCache(): void
+  someNewlyAddedBreakingMethod(): void
```

Comment:
That's a cool test, thanks for going these extra rounds, make things very concrete and explicit! 🙂